compiler/options.nim

```diff
@@ -524,18 +525,19 @@ proc getOutFile*(conf: ConfigRef; filename: RelativeFile, ext: string): Absolute
   conf.outDir / changeFileExt(filename, ext)
 
 proc absOutFile*(conf: ConfigRef): AbsoluteFile =
+  if false:
```

Comment:
if (instead of when) intentional to ensure body keeps compiling; i'll fix this soon anyways in other PR

Reply:
Alright...

Reply:
=> fixing this in https://github.com/nim-lang/Nim/pull/14555